<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortMediaDialog</class>
 <widget class="QDialog" name="SortMediaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-180</x>
     <y>130</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>178</width>
     <height>124</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="sortingCriteriaLabel">
        <property name="text">
         <string>sorting criteria:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="typeRadioButton">
        <property name="text">
         <string>type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="availabilityRadioButton">
        <property name="text">
         <string>availability</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="borrowerRadioButton">
        <property name="text">
         <string>borrower</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="titleRadioButton">
        <property name="text">
         <string>title</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="sortingOrderLabel">
        <property name="text">
         <string>sorting order:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="descendingCheckBox">
        <property name="text">
         <string>descending</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SortMediaDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SortMediaDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
